documentation/c11-xmlproc.sil

```diff
@@ -6,29 +6,29 @@ We’ll be looking at the DocBook processor that ships with SILE.
 DocBook is an XML format for structured technical documentation.
 DocBook itself doesn’t encode any presentation information about how its various tags should be rendered on a page, and so we shall have to make all the presentation decisions for ourself.
 
-Since DocDook itself doesn’t specify anything about presentation such as papersize, you may need to supply values either on the command line or using a preamble.
-When you use the \code{-c docbook} command line option to SILE, SILE will use the \em{docbook} class in spite of any document declaration.
-In addition options such as paper size could be set with, e.g. \code{-O papersize=legal}.
+Since DocDook itself doesn’t specify anything about presentation such as paper size, you may need to supply values either on the command line or using a preamble.
+When you use the \code{-c docbook} command line option to SILE, SILE will use the \code{docbook} class in spite of any document declaration.
+In addition, options such as paper size could be set; for example, \code{-O papersize=legal}.
 
-The class initalization for DocBoox isn’t too fancy, it just loads up a couple packages that will get used later.
-
-Now we can start defining SILE commands to render XML elements.
-Most of these are fairly straightforward so we will not dwell on them too much.
-For instance, DocBook has tags like \code{<code>}, \code{<filename>}, \code{<guimenu>} which should all be rendered in a monospaced typewriter font.
-To make it easier to customize the class, we abstract out the font change into a single command:
+The class initalization for DocBoox isn’t too fancy; it just loads up a couple packages that will get used later.
 
 \begin{note}
-Much of the example code in this chapter is in SIL format using macros.
-The actual docbook class currently uses Lua functions to specify these commands.
+Much of the example code in this chapter is in SILE format using macros.
```

Comment:
SIL / SILE format: I think it was discussed somewhere (@alerque might certainly remember) on how to call the TeX-like input format, and SIL was the choice (to distinguish that input format from SILE as a software that can use other input formats, IIRC). Might not be very clear, though.

Reply:
Yes, the file type and input language should be SIL not SILE. I touched this up. We probably have lots more places in need of an update since this was a change in early development.